doc/man1/flux-mini.rst

```diff
@@ -394,6 +395,10 @@ OTHER OPTIONS
    Enable job debug events, primarily for debugging Flux itself.
    The specific effects of this option may change in the future.
 
+**--bg**
+   *(alloc only)* Do not interactively attach to the instance. Instead,
+   print jobid on stdout once the instance is ready to accept jobs.
+
```

Comment:
Is it worth noting that in this mode, `flux job attach` cannot be used to interactively attach later?

Reply:
Yes, good idea. 

Some background:

I actually had the command starting an interactive shell at first so that users could later `flux job attach` to their job. In this case, you can still cancel the job or tell it to shutdown and the shell will get SIGHUP or killed and the job will exit. However, the jobs then are recorded as FAILED.

With `broker.rc2_none` set, it seemed like a nice side effect that you can `flux shutdown $jobid` and the job will complete successfully instead of failing.

Reply:
That makes good sense.  Maybe it's useful to mention that `flux shutdown $jobid` is the expected way the instance should be stopped then?

Reply:
Yeah, probably. 

Though that command issues a mass of broker log messages on stdout by default, I was thinking we should suppress that by default without -v if normal users were going to run it. I wasn't sure what you thought of that in the typical case of a sysadmin using it on the system instance though.

Reply:
What about suggesting `flux shutdown --quiet`?

I'd be a bit worried about changing the default on that one just because if something hangs, it's nice to know where in the process you are.  I guess I'm primarily concerned about system instances there.

Reply:
Yeah, I was worried about changing default behavior as well. It isn't a big enough problem to worry about at the moment.